packet_pool.go

```diff
@@ -5,17 +5,74 @@ import (
 	"sync"
 )
 
-// packetPool represents a pool of packets
+// packetAccumulator keeps track of packets for a single PID and decides when to flush them
+type packetAccumulator struct {
+	pid  uint16
```

Comment:
Could you reorder fields alphabetically here?

Reply:
sure